modules/core-updates/9.0.md

```diff
@@ -58,6 +58,15 @@ We used this occasion to clean the dependencies that are no longer used in the c
 - `symfony/uid`
 - `symfony/workflow`
 
+{{% notice note %}}
+**Mail SSL encryption was dropped**
+
+When we were migrating Swift mailer to Symfony Mailer, we noticed that ssl support was not an option in ESMTP transport. Also SSL is an old and outdated encryption type. For security reasons, it will not be allowed anymore.
```

Comment:
```suggestion
When we migrated from Swift Mailer to Symfony Mailer, we noticed that SSL support was not an option in ESMTP transport. SSL is an old and outdated encryption type, and for security reasons, it will no longer be allowed.
```